src/validation.cpp

```diff
@@ -331,9 +331,7 @@ void CChainState::MaybeUpdateMempoolForReorg(
     while (it != disconnectpool.queuedTx.get<insertion_order>().rend()) {
         // ignore validation errors in resurrected transactions
         if (!fAddToMempool || (*it)->IsCoinBase() ||
-            AcceptToMemoryPool(*m_mempool, *this, *it, GetTime(),
-                /* bypass_limits= */true, /* test_accept= */ false).m_result_type !=
-                    MempoolAcceptResult::ResultType::VALID) {
+            AcceptToMemoryPool(*this, *it, GetTime(), /* bypass_limits= */true, /* test_accept= */false).m_result_type != MempoolAcceptResult::ResultType::VALID) {
```

Comment:
I don't understand why you keep changing the formatting on this line, where all you need to do is remove the `m_mempool` argument. 160 columns is too wide for many people to do a side-by-side diff.

Reply:
Done in 2cc5a38